{"name": "test_unsuccessfull_login_courier_without_login_field", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'Недостаточно...ных для входа' == 'Недостаточно...четной записи'\n  \n  \u001b[0m\u001b[91m- Недостаточно данных для создания учетной записи\u001b[39;49;00m\u001b[90m\u001b[39;49;00m\n  \u001b[92m+ Недостаточно данных для входа\u001b[39;49;00m\u001b[90m\u001b[39;49;00m", "trace": "self = <tests.test_login_courier.TestLoginCourier object at 0x00000259608C4770>, сreate_courier = ['eturlryqiw', 'zvusonxarp', 'hlyxavnigv']\n\n    def test_unsuccessfull_login_courier_without_login_field(self, сreate_courier):\n            with allure.step(\"формирование тела запроса\"):\n                payload = {\n                    \"password\": сreate_courier[1]\n                }\n    \n            with allure.step(\"отправляем запрос на логин и сохраняем ответ в переменную response\"):\n                response = requests.post(Path.LOGIN, data=payload)\n            with allure.step(\"проверка кода и тела ответа\"):\n                assert response.status_code == 400\n>               assert response.json()[\"message\"] == \"Недостаточно данных для создания учетной записи\"\nE               AssertionError: assert 'Недостаточно...ных для входа' == 'Недостаточно...четной записи'\nE                 \nE                 \u001b[0m\u001b[91m- Недостаточно данных для создания учетной записи\u001b[39;49;00m\u001b[90m\u001b[39;49;00m\nE                 \u001b[92m+ Недостаточно данных для входа\u001b[39;49;00m\u001b[90m\u001b[39;49;00m\n\ntests\\test_login_courier.py:46: AssertionError"}, "steps": [{"name": "формирование тела запроса", "status": "passed", "start": 1757337511909, "stop": 1757337511909}, {"name": "отправляем запрос на логин и сохраняем ответ в переменную response", "status": "passed", "start": 1757337511909, "stop": 1757337512257}, {"name": "проверка кода и тела ответа", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'Недостаточно...ных для входа' == 'Недостаточно...четной записи'\n  \n  \u001b[0m\u001b[91m- Недостаточно данных для создания учетной записи\u001b[39;49;00m\u001b[90m\u001b[39;49;00m\n  \u001b[92m+ Недостаточно данных для входа\u001b[39;49;00m\u001b[90m\u001b[39;49;00m\n", "trace": "  File \"c:\\Users\\Dmitry\\Desktop\\Python YP\\Sprint_7\\tests\\test_login_courier.py\", line 46, in test_unsuccessfull_login_courier_without_login_field\n    assert response.json()[\"message\"] == \"Недостаточно данных для создания учетной записи\"\n"}, "start": 1757337512257, "stop": 1757337512257}], "start": 1757337511909, "stop": 1757337512258, "uuid": "2ca2cada-2f37-48ad-96b5-df0f1c108286", "historyId": "cb98a2696d0456f1475a74264a44f2ff", "testCaseId": "cb98a2696d0456f1475a74264a44f2ff", "fullName": "tests.test_login_courier.TestLoginCourier#test_unsuccessfull_login_courier_without_login_field", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "DESKTOP-A445VSF"}, {"name": "thread", "value": "14248-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}], "titlePath": ["tests", "test_login_courier.py", "TestLoginCourier"]}